roottest/root/treeformula/parse/ClassDefinitions.C

```diff
@@ -10,6 +10,7 @@ class QRawPulseR : public TObject
 		QRawPulseR();
 		~QRawPulseR() override;
 		const QRawPulseR& operator=(const QRawPulseR& P);
+		QRawPulseR(const QRawPulseR& P) : fDataHist(P.DataHist), fChannel(P.fChannel) {}
```

Comment:
Doesn't
```suggestion
		QRawPulseR(const QRawPulseR& P) = default;
```
work as well?